src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -59,6 +59,12 @@ void CCrossValidation::init()
 	SG_ADD((CSGObject**)&m_xval_outputs, "m_xval_outputs", "List of output "
 			"classes for intermediade cross-validation results",
 			MS_NOT_AVAILABLE);
+
+	/* Register observable paramaters of CrossValidation */
+	register_observable_param(
+	    "fold_result_0", "scalar",
+	    "Intermediate fold evaluation results. There will be many parameters "
```

Comment:
Don't think this "there will be many" is instructive. Rather name them self-exlaining